src/insert-expand-diff/insert-expand-diff.js

```diff
@@ -0,0 +1,39 @@
+// @flow
+// @jsx h
+
+import { h } from 'dom-chef'
+
+const onClick = (diff: HTMLElement) => {
+    const expandElipses = diff.querySelectorAll('.ellipsis')
+
+    if (!expandElipses) return
+
+    expandElipses.forEach(expander => expander.click())
+}
+
+export default function insertExpandDiff(diff: HTMLElement) {
+    const button = (
+        <button
+            type="button"
+            class="aui-button aui-button-subtle devtools-task-in-progress--button __rbb-btn-expandDiff"
```

Comment:
Is this `devtools-task-in-progress--button` CSS class available in Firefox? Does the button look as intended in Firefox ?

Reply:
I tested this in chrome and firefox, and it looked fine in both.  The truth is I got these classnames from the atlassian documentation recommended when reusing their imageset.  I've attempted to find this exact string again, and cannot find it anywhere on any atlassian documentation page.  Atlassian documentation isn't the easiest thing to navigate on the best of days, but I think it might have genuinely changed since last October, regardless.